extensions/extensions-helm/aissemble-inference-chart/README.md

```diff
@@ -11,9 +11,9 @@ helm install inference ghcr.io/boozallen/aissemble-inference-chart --version <AI
 # Properties
 | Property                                | Description                                                                                                                      | Required Override | Default                                                                                                |
 |-----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------|--------------------------------------------------------------------------------------------------------|
- | app.name                                | Sets label for app.kubernetes.io/name                                                                                            | No                | Chart.Name (aissemble-inference-chart)                                                                 |
+| app.name                                | Sets label for app.kubernetes.io/name                                                                                            | No                | Chart.Name (aissemble-inference-chart)                                                                 |
 | app.version                             | Sets label for app.kubernetes.io/version                                                                                         | No                | Chart.AppVersion (aiSSEMBLE project version)                                                           |
-| deployment.image.repo                   | The image repository                                                                                                             | No                | NB: OSS: update with default aissemble docker repository                                               | 
+| deployment.image.repo                   | The image repository                                                                                                             | No                | docker-registry-PLACEHOLDER/repository/ (default if no custom repo is specified)                       | 
```

Comment:
C: This is the documentation for the chart itself, outside the context of an aissemble-based project.  It should reflect the value set in the default `values.yaml` file that is a sibling to this README.  Same is true for other chart READMEs in this PR.

Reply:
Ah gotcha. So this should be updated to ghcr.io right?

Reply:
I'd have to check the values, but yes that sounds correct.

Reply:
Updated READMEs in PR to reference the repo in the values file (ghcr.io)